### PR TITLE
[ticket/14624] Add event to ucp_profile in signature section

### PR DESCRIPTION
**Pull request by rmcgirr83**
https://tracker.phpbb.com/browse/PHPBB3-14624

PHPBB3-14624

### CODE REVIEW

**Comment by RMcGirr83** [comment]
!set Event

File: phpBB/includes/ucp/ucp_profile.php [hunk 1]
```diff
@@ -492,6 +492,32 @@ function main($id, $mode)
...
+						* @event core.ucp_profile_modify_signature
+						* @var	bool	enable_bbcode		Whether or not bbcode is enabled
+						* @var	bool	enable_smilies		Whether or not smilies are enabled
+						* @var	bool	enable_urls			Whether or not urls are enabled
+						* @var	string	signature			Users signature text
+						* @var	object	message_parser		The message parser object
+						* @var	array	error				Any error strings
+						* @var	bool	submit				Whether or not the form has been sumitted
+						* @var	bool	preview				Whether or not the signature is being previewed
+						* @since 3.1.9
```
Comment (marc1706) [inline phpBB/includes/ucp/ucp_profile.php]: Needs to be 3.1.10-RC1

File: phpBB/includes/ucp/ucp_profile.php [hunk 2]
```diff
@@ -492,6 +492,32 @@ function main($id, $mode)
...
+						*/
+						$vars = array(
+							'enable_bbocde',
```
Comment (marc1706) [inline phpBB/includes/ucp/ucp_profile.php]: Typo

**Comment by RMcGirr83** [comment]
Thanks @marc1706.  Made the changes and moved the event so it would actually do something :eyes:

**Pull request merged**